### PR TITLE
New v3.15.1 of Gatekeeper.

### PR DESCRIPTION
**Pull request by max-allan-cgr**
Gatekeeper-3.15: new version of Gatekeeper package

Fixes:
https://github.com/chainguard-dev/customer-issues/issues/1332

#### For new package PRs only

- [x] This PR is marked as fixing a pre-existing package request bug
- [x] REQUIRED - The package is available under an OSI-approved or FSF-approved license
- [x] REQUIRED - The version of the package is still receiving security updates

"This PR links to the upstream project's support policy (e.g. `endoflife.date`)"

The project has no support policy I can find.

NB, the "gator" sub pkg doesnt appear in "apk search" results for v3.14. The 3.15 gator does. Don't know if that's my problem or the 3.14 pkg's problem or not a problem.
```
6066cd522060:/work/packages# apk search gatekeeper | sort
gatekeeper-3.12-3.12.0-r14
gatekeeper-3.12-compat-3.12.0-r14
gatekeeper-3.13-3.13.4-r5
gatekeeper-3.13-compat-3.13.4-r5
gatekeeper-3.13.0-r1
gatekeeper-3.14-3.14.1-r2
gatekeeper-3.14-compat-3.14.1-r2
gatekeeper-3.15-3.15.1-r0
gatekeeper-3.15-compat-3.15.1-r0
gatekeeper-3.15-gator-3.15.1-r0
gatekeeper-compat-3.13.0-r1
```

### CODE REVIEW

File: gatekeeper-3.15.yaml [hunk 1]
```diff
@@ -0,0 +1,68 @@
+package:
+  name: gatekeeper-3.15
```
Comment (jamonation) [inline gatekeeper-3.15.yaml:2]:
```suggestion
  name: gatekeeper
```

File: gatekeeper-3.15.yaml [hunk 2]
```diff
@@ -0,0 +1,68 @@
...
+      provides:
+        - gatekeeper-gator=${{package.full-version}}
+
+update:
+  enabled: true
+  github:
+    identifier: open-policy-agent/gatekeeper
+    strip-prefix: v
+    use-tag: true
+    tag-filter: v3.15.
```
Comment (jamonation) [inline gatekeeper-3.15.yaml:68]:
```suggestion
```

**Review by jamonation — COMMENTED** [review]
I think all this needs is a rename of `gatekeeper-3.15.yaml` -> `gatekeeper.yaml` and the small changes to the package name/tag version and it should be good to go!

**Comment by rawlingsj** [comment]
Thanks Max - we now have [version stream data for gatekeeper](https://github.com/chainguard-dev/package-version-metadata/blob/main/version_streams/gatekeeper.yaml) so we've had the automated PRs create this now https://github.com/wolfi-dev/os/pull/17324

I'm going to close this as I think all is working now.

**Comment by rawlingsj** [comment]
Thanks Max - we now have [version stream data for gatekeeper](https://github.com/chainguard-dev/package-version-metadata/blob/main/version_streams/gatekeeper.yaml) so we've had the automated PRs create this now https://github.com/wolfi-dev/os/pull/17324

I'm going to close this as I think all is working now.

**Pull request closed without merging**